model/database/registry.yaml

```diff
@@ -555,6 +558,32 @@ groups:
           in [Azure Location API](https://learn.microsoft.com/rest/api/subscription/subscriptions/list-locations?view=rest-subscription-2021-10-01&tabs=HTTP#location)
         examples:
           - ["North Central US", "Australia East", "Australia Southeast"]
+
+  - id: registry.db.cosmosdb.network
+    type: attribute_group
+    display_name: Azure Cosmos DB Network level Attributes
+    stability: experimental
+    brief: >
+      This group defines attributes for Azure Cosmos DB Network call.
+    attributes:
+      - id: db.cosmosdb.network.response.status_code
+        type: int
+        stability: experimental
+        brief: The status code returned in response.
+        note: The protocol reuses HTTP status codes.
```

Comment:
I think it would be good to explain the difference between `db.cosmosdb.network.response.status_code` and `db.response.status_code` here.